Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -377,7 +377,6 @@ open class _ChatMessageContentView<ExtraData: ExtraDataTypes>: _View, ThemeProvi
             let content = content,
             content.imageAttachments.isEmpty,
             content.giphyAttachments.isEmpty,
-            content.linkAttachments.isEmpty,
             !content.linkAttachments.isEmpty {
             bubbleView?.backgroundColor = appearance.colorPalette.highlightedAccentBackground1
```

Comment:
I think it's safe to remove this logic from here and rather implement changing the background color in the link attachment injector. It will also solve the edge case with multiple attachments with different types.

Reply:
Btw here I just fixed, a typo. `content.linkAttachments.isEmpty && !content.linkAttachments.isEmpty` would never work. Do you think we should move all this logic to each injector:
<img src="https://user-images.githubusercontent.com/12814114/118797115-26c44680-b894-11eb-82e4-9c40060fc15e.png" width="40%">